src/test/scala/uk/gov/nationalarchives/tdr/api/db/repository/ConsignmentRepositorySpec.scala

```diff
@@ -232,6 +233,46 @@ class ConsignmentRepositorySpec extends TestContainerUtils with ScalaFutures wit
       response should have size 0
   }
 
+  "getConsignments" should "return all the consignments which assigned to the given user id only" in withContainers {
```

Comment:
I would reword the test slightly to: `... consignments which belong to the given user id ...`